README.md

```diff
@@ -77,7 +77,99 @@ help xtevent
 
 -----------
 
+### Examples
+
+Using xtevent 2.1.0
+#### xtevent
+```stata
+*setup
+webuse nlswork
+xtset idcode year
+
+*Estimate a basic event study with clustered standard errors. 
+*Besides, impute the policy variable without verifying staggered adoption.
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(3) cluster(idcode) impute(nuchange)
+            
+*Omit fixed effects
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(3) cluster(idcode) impute(nuchange) nofe note
+
+*Adjust by estimating a linear trend with gmm method
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(2) cluster(idcode) impute(nuchange) trend(-2, ///
+            method(gmm))
+      
+*FHS estimator with proxy variables
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(3) vce(cluster idcode) impute(nuchange) ///
+            proxy(wks_work)
+
+*reghdfe and two-way clustering
+xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
+            pol(union) w(3) impute(nuchange) cluster(idcode year) reghdfe ///
+            proxy(wks_work)
+
+
+```
+#### xteventplot
+```
+*setup
+webuse nlswork
+xtset idcode year
+
+*Add an extra effect if union equals 1

```

Comment:
Why do we need this extra union effect?

Reply:
@jorpppp with the original `ln_wage` variable the plot didn't show any effect in the post-treatment coefficients, and when specifying the `smpath` option in the next examples, the plotted smoothest path was just a horizontal line as shown here:
![1 ](https://user-images.githubusercontent.com/36095845/184049324-8af3e0a8-ada7-48ef-916e-d560fd601fff.png)

but adding the extra value, the smoothest path plot is the following:
![2](https://user-images.githubusercontent.com/36095845/184049417-3bd6e0e7-e0ea-4ea0-be72-bab7903f7e28.png)

Reply:
@Constantino-Carreto-Romero got it, thanks.